core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -57,4 +75,88 @@ public boolean containsKey(Object k) {
    public void evict(WrappedBytes key) {
       super.evict(0, key);
    }
+
+   @Override
+   public void put(WrappedBytes key, WrappedBytes value, Metadata metadata) {
```

Comment:
I wish this wasn't required, that we could reuse `AbstractInternalDataContainer` instead. But unfortunately these classes were written assuming they wouldn't be called in the wrong context, so I am fine with it.